processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -338,7 +492,7 @@ public void testGroupByWithSimpleRename()
         .setDimensions(
             Lists.<DimensionSpec>newArrayList(
                 new ExtractionDimensionSpec(
-                    "quality", "alias", new LookupExtractionFn(new MapLookupExtractor(map), false, null, true), null
+                    "quality", "alias", new LookupExtractionFn(new MapLookupExtractor(map), false, "MISSING", true), null
```

Comment:
This is the test that would have caught the bug for TopN. From a testing perspective, if the second parameter (retain missing values) is `false` both `null` and `"MISSING"` would work here, but if I change the second parameter to `true` (which is what caused the bug to be missed in topN) then only `null` is valid. `"MISSING"` will fail validation.

Essentially, if anyone ever copies around this test for more stuff, it makes it less likely to accidentally flip a parameter.